docs/containertools/README.md

```diff
@@ -36,7 +36,7 @@ management, monitoring, logging and resource isolation as required by CRI.
 
 ## Podman
 
-Podman is a command line tool that resides in the [libpod](https://github.com/containers/libpod) project.
+Podman is a command line tool that resides in the [libpod](https://github.com/containers/podman) project.
```

Comment:
I think we can delete this line (as it's referring libpod) and add the link to the first occurrence of "Podman" below?